src/ReplenishFunction.ts

```diff
@@ -56,6 +56,7 @@ async function defaultReplenishFunction(
     if (mustReplenishWorker && !isReplenishPendingForWorker) {
         const refill = toBN(
             relayServer.config.workerTargetBalance.toString()
+            // @ts-ignore
```

Comment:
These are needed since the `bn.js` types are not well defined, the usage is ok but the types checks are failing.

Reply:
removed